From 1ac0ee2bae7fc79d7be922832de67f9df68f635f Mon Sep 17 00:00:00 2001
From: Davide Caratti <dcaratti@redhat.com>
Date: Tue, 12 Dec 2017 16:45:15 +0100
Subject: [PATCH] tc: bash-completion: add missing 'classid' keyword

users of 'matchall' filter can specify a value for the class id: update
bash-completion accordingly.

Fixes: b32c0b64fa2b ("tc: bash-completion: Add support for matchall")
Signed-off-by: Davide Caratti <dcaratti@redhat.com>
(cherry picked from commit 88b428f03f7c1fa64331105f54d42806cece77be)
---
 bash-completion/tc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/bash-completion/tc b/bash-completion/tc
index 80d12972c8f3c..29bca5d9354d3 100644
--- a/bash-completion/tc
+++ b/bash-completion/tc
@@ -450,7 +450,7 @@ _tc_filter_options()
             return 0
             ;;
         matchall)
-            _tc_once_attr 'action skip_sw skip_hw'
+            _tc_once_attr 'action classid skip_sw skip_hw'
             return 0
             ;;
         flower)
-- 
2.13.1

